Add log for debugging

leepengxiang opened the pull request.

Commit:
Add log for debugging

bogdandrutu commented on exporter/exporterwrapper/exporterwrapper.go:
```diff
@@ -88,6 +96,10 @@ func PushOcProtoSpansToOCTraceExporter(ocExporter trace.Exporter, td data.TraceD
 	}
+	log.Printf("spans: %d\tgoogle spans: %d\n", len(td.Spans), len(goodSpans))
+	log.Printf("https://pantheon.corp.google.com/traces/traces?project=cloud-debugging&tid=%x%x\n",
+		binary.LittleEndian.Uint64(goodSpans[0].TraceId[0:8]),
```
They do BigEndian.

leepengxiang replied:
Done.

bogdandrutu commented on exporter/exporterwrapper/exporterwrapper.go:
```diff
@@ -80,6 +82,12 @@ func PushOcProtoSpansToOCTraceExporter(ocExporter trace.Exporter, td data.TraceD
+			binary.LittleEndian.Uint64(span.TraceId[0:8]),
+			binary.LittleEndian.Uint64(span.TraceId[8:16]),
+			binary.LittleEndian.Uint64(span.SpanId),
+			binary.LittleEndian.Uint64(span.ParentSpanId),
```
Print nil here probably if ParentSpanId is nil.

leepengxiang replied:
Done.

Commit:
Advance the end time of a span by 1ns if end time is the same as start
time. Span with 0 duration won't show in StackDriver.
Also use big-endian to log out the trace ID for debugging.

Commit:
Clean the redundant code for calculate start and end time for spans.

Commit:
Minor fix on typo
